packages/chakra-components/docs/03-Election.mdx

```diff
@@ -36,21 +36,33 @@ Show election title.
 
 ## Election Schedule
 
-Show election period. You can specify date format using [date-fns](https://date-fns.org/v2.30.0/docs/format) format.
+Show election period. You can specify date format using [date-fns](https://date-fns.org/v2.30.0/docs/format) format
 
 ```jsx live
 <ElectionSchedule format={'d-L-y HH:mm'} />
 ```
 
 ## Election Status Badge
 
-It shows election status badge. Possible values of the election are defined on `ElectionStatus` from `@vocdoni/sdk`:
+It shows election status badge. Possible values of the election are defined on `ElectionStatus` from [`@vocdoni/sdk`](https://github.com/vocdoni/vocdoni-sdk/blob/main/src/types/election/published.ts#L6-L18):
 `PROCESS_UNKNOWN`, `UPCOMING`, `ONGOING`, `ENDED`, `CANCELED`, `PAUSED`, `RESULTS`.
 
 ```jsx live
 <ElectionStatusBadge variant='solid' colorScheme='teal' />
 ```
 
+To style the [`ElectionStatusBadge`](https://chakra-ui.com/docs/components/tag) leverage the capabilities of the Chakra
+styled system. By simply adjusting the `colorScheme` property based on the election status will enable you to easily alter the
+color of the badge. This approach mirrors the one employed by the
+[default color scheme](https://github.com/vocdoni/ui-components/blob/main/packages/chakra-components/src/components/Election/StatusBadge.tsx#L12-L24)
+included on the Vocdoni Chakra Components.
+
+For a more comprehensive customization, consider creating a custom variant as demonstrated in the
+[chakra docs](https://chakra-ui.com/docs/components/tag/theming#adding-a-custom-variant).
+
+Furthermore, choosing to use the `ElectionStatusBadge` instead of directly implementing the logic for the `Tag` Chakra
+component brings the added benefit of pre-included translations.
```

Comment:
Same as before... in this case I'd remove this line. chakra-components actually do not use translations, you have to provide them to the ClientProvider (using the `locale` prop), so, in the end, if you need to overwrite it and are using translations, it will probably take less time to just replace the entire component with one of your own.

Reply:
Force pushed fix bfc1aa7